$NetBSD: patch-am,v 1.1 2001/03/23 16:20:56 agc Exp $

Patch from Wolfgang Solfrank:

Disabling the O_SYNC option when opening printer or punch.

--- cardpch.c.orig	Sun Jan 28 18:05:38 2001
+++ cardpch.c	Fri Mar 23 16:01:53 2001
@@ -155,9 +155,9 @@
     {
         rc = open (dev->filename,
 #ifdef WIN32
-                    O_WRONLY | O_CREAT | O_TRUNC | O_SYNC | O_BINARY,
+                    O_WRONLY | O_CREAT | O_TRUNC | O_BINARY,
 #else /* WIN32 */
-                    O_WRONLY | O_CREAT | O_TRUNC | O_SYNC,
+                    O_WRONLY | O_CREAT | O_TRUNC,
 #endif /* WIN32 */
                     S_IRUSR | S_IWUSR | S_IRGRP);
         if (rc < 0)
